srcpkgs/ghc/template

```diff
@@ -2,7 +2,7 @@
 pkgname=ghc
 # Keep this synchronized with http://www.stackage.org/lts
 version=8.8.4
-revision=2
+revision=3
```

Comment:
If this is only to fix the build, I don't think we need to revbump.

Reply:
Fair enough - anyone having a previously  compiled version is not likely interested to have a new revision forced onto them.  Would I need to push an updated commit?

Reply:
Yes, please.